Packs/CloudConvert/Integrations/CloudConvert/CloudConvert.py

```diff
@@ -0,0 +1,396 @@
+        results_info = results.get('data', {}).get('result', {}).get('files', [{}])[0]
+        url = results_info.get('url')
+        file_name = results_info.get('filename')
+        file_data = client.get_file_from_url(url)
+        war_room_file = fileResult(filename=file_name, data=file_data)
+        readable_output = tableToMarkdown('Check Status Results', remove_empty_elements(results_data),
+                                          headers=('created_at', 'depends_on_task_ids', 'id', 'operation', 'result',
+                                                   'status'), headerTransform=string_to_table_header)
+        return_results(CommandResults(
+            outputs_prefix='CloudConvert.Task',
+            outputs_key_field='id',
+            raw_response=results,
+            readable_output=readable_output,
+            outputs=remove_empty_elements(results_data)
+        ))
+        return war_room_file
+
+    else:
+        readable_output = tableToMarkdown('Check Status Results', remove_empty_elements(results_data),
+                                          headers=('created_at', 'depends_on_task_ids', 'id', 'operation', 'result',
+                                                   'status'),
+                                          headerTransform=string_to_table_header)
+        return CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='CloudConvert.Task',
+            outputs_key_field='id',
+            raw_response=results,
+            outputs=remove_empty_elements(results_data)
+        )
+
+
+@logger
+def export_command(client: Client, arguments: Dict[str, Any]):
+    """
+    Export a converted file back to user, either as a url or directly to a war room entry
+    Note: in order to get the resulted url/entry of the file you need to use a check-status command as well,
+        since the response of the export command is usually responded before the file is fully exported (hence the
+    'status' field is 'waiting', and not 'finished')
+    :param client: CloudConvert client to use
+    :param arguments: All command arguments, the fields 'task_id', and 'export_as' (url/war_room_entry)
+    :return: CommandResults object containing the results of the export action as returned from the API, and its readable
+     output.
+    """
+    # Call export to url request
+    # In both url and war room entry we still first get a url
+    results = client.export_url(arguments)
+    results_data = results.get('data')
+
+    # No 'data' field was returned from the request, meaning the input was invalid
+    if results_data is None:
+        if results.get('message'):
+            raise ValueError(results.get('message'))
+        else:
+            raise ValueError('No response from server, the server could be temporary unavailable or it is handling too '
+                             'many requests. Please try again later.')
```

Comment:
code duplication across all command functions - move it to another function (like the requests module has a `raise_for_status` function).